test/runner.js

```diff
@@ -18,37 +18,49 @@ const debug = require( 'debug' )( 'test-runner' ),
  * Internal dependencies
  */
 const boot = require( './boot-test' ),
+	utils = require( './utils.js' ),
 	setup = require( './setup' );
 
 program
 	.usage( '[options] [files]' )
 	.option( '-R, --reporter <name>', 'specify the reporter to use', 'spec' )
 	.option( '-t, --node-total <n>', 'specify the node total to use', parseInt )
 	.option( '-i, --node-index <n>', 'specify the node index to use', parseInt )
-	.option( '-g, --grep <pattern>', 'only run tests matching <pattern>' );
+	.option( '-g, --grep <pattern>', 'only run tests matching <pattern>' )
+	.option( '-w, --watch', 'watch files for changes' );
 
 program.name = 'runner';
 
 program.parse( process.argv );
 
-const mocha = new Mocha( {
-	ui: 'bdd',
-	reporter: program.reporter
-} );
+let getMocha = function() {
```

Comment:
We encourage `const` over `let` unless you're reassigning the value ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations))